include/frame.h

```diff
@@ -347,12 +347,12 @@ class FrameDiff {
 
 // These diffing functions will order the IDs of units in each frame, and thus
 // is not const.
-FrameDiff frame_diff(Frame& lhs, Frame& rhs);
-FrameDiff frame_diff(Frame* lhs, Frame* rhs);
-Frame* frame_undiff(FrameDiff& lhs, Frame& rhs);
-Frame* frame_undiff(Frame& lhs, FrameDiff& rhs);
-Frame* frame_undiff(FrameDiff* lhs, Frame* rhs);
-Frame* frame_undiff(Frame* lhs, FrameDiff* rhs);
+FrameDiff frame_diff(Frame&, Frame&);
+FrameDiff frame_diff(Frame*, Frame*);
+Frame* frame_undiff(FrameDiff*, Frame*);
+Frame* frame_undiff(Frame*, FrameDiff*);
+void frame_undiff(Frame*, FrameDiff*, Frame*);
+void frame_undiff(Frame*, Frame*, FrameDiff*);
```

Comment:
It would be nice to provide `from` and `to` argument names here to emphasize the direction of the diff.